README.md

```diff
@@ -508,6 +509,46 @@ testcases:
 
 ```
 
+## Iterating over data
+
+It is possible to iterate over data using `range` attribute.
+
+The following data types are supported, each exposing contexted variables `.index`, `.key` and `.value`:
+
+- An array where each value will be iterated over (`[]interface{}`)
+  - `.index`/`.key`: current iteration index
+  - `.value`: current iteration item value
+- A map where each key will be iterated over (`map[string]interface{}`)
+  - `.index`: current iteration index
+  - `.key`: current iteration item key
+  - `.value`: current iteration item value
+- An integer to perform target step `n` times (`int`)
+  - `.index`/`.key`/`.value`: current iteration index
+- A templated string which results in one of the above typing (`string`)
+  - It can be either inherited from vars file, or interpolated from a previous step result
+
+Within a range context, `.range` contexted variable will be set to `true` (while it'll be `false` in regular contexts)
```

Comment:
in fact i'm not sure the `{{.range}}` is still useful ? how do you think you will use it ?

Reply:
same as above, it's mostly for unit test to check in which context the step is running

Reply:
i guess the other tests can assert the whole feature is working. i prefer to remove it if it's not strictly required.

Reply:
ok I removed all references to `{{.range}}` and also removed the go code that was setting it